BitwardenShared/Core/Tools/Repositories/GeneratorRepository.swift

```diff
@@ -164,6 +170,17 @@ extension DefaultGeneratorRepository: GeneratorRepository {
 
     // MARK: Generator
 
+    func generateMasterPassword() async throws -> String {
+        try await clientService.generators().passphrase(
+            settings: PassphraseGeneratorRequest(
+                numWords: 3,
```

Comment:
❓ Is product ok with this being fixed at `3`? I'd use 4 (or customizable by the user) for a stronger master password but well 3 is easier to remember...

Reply:
Per the figma design it was 3.  I can confirm but I believe when Shannon did this work she was going off the figma doc that was provided.  

<img width="273" alt="Screenshot 2024-09-30 at 2 43 39 PM" src="https://github.com/user-attachments/assets/46927c60-771d-41a2-93df-2410aa46dd2e">

Reply:
Confirmed that Android is using 3 words for their implementation.